packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md

```diff
@@ -0,0 +1,263 @@
+---
+title: Connect your SCIM API service to Okta
+excerpt: Create a SCIM integration directly within Okta, configure the SCIM options for that integration, and check the attributes and corresponding mappings.
+meta:
+  - name: description
+    content: Create a SCIM integration directly within Okta, configure the SCIM options for that integration, and check the attributes and corresponding mappings.
+layout: Guides
+---
+
+This guide — part of the [Build a SCIM provisioning integration](/docs/guides/scim-provisioning-integration-overview/) series — teaches you how to create a SCIM integration directly within Okta, configure the SCIM options for that integration, and check the attributes and corresponding mappings.
+
+## Getting connected
+
+After you have a SCIM implementation that passes all of the Runscope tests, as discussed in the [previous part of the series](/docs/guides/scim-provisioning-integration-prepare/main/#test-your-scim-api), you need to create your SCIM integration directly within Okta.
+
+Begin by signing up for an [Okta developer account](https://developer.okta.com/signup/).
+
+1. After you request the developer account and have received the initial email, click the link to go to your Admin Console.
+1. In the Admin Console, go to **Applications** > **Applications**.
```

Comment:
"In the Admin Console" is redundant as the step before tells them they are already there. How about:
1. Select **Applications** > **Applications**.